haas/api.py

```diff
@@ -138,9 +137,8 @@ def project_detach_node(project, node):
     project.nodes.remove(node)
     db.session.commit()
 
-
-                            # Node Code #
-                            #############
+    # Node Code #
+    #############
```

Comment:
If we're going to left-align all of these, this should probably not be indented at all, like the others.